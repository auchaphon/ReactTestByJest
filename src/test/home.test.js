import React from "react";
import { shallow } from "enzyme";
import Home from "../components/home/index.js";

it("Should be find element correctly ", () => {
  const props = {
    header: "Auchaphon",
    body: "Auchaphon Body"
  };
  const wrapper = shallow(<Home {...props} />);
  expect(wrapper).toMatchSnapshot();
});
