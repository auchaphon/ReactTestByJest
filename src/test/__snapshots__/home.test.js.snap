// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should be find element correctly  1`] = `
<div>
  <h1
    id="header"
  >
    Auchaphon
  </h1>
  <p
    id="body"
  >
    Auchaphon Body
  </p>
</div>
`;
